docs/etcd/cluster_setup.md

```diff
@@ -0,0 +1,253 @@
+# Set up an etcd cluster
+
+This guide will help you set up an example etcd cluster that you can use with CN-WAN Operator.
+
+In the first part, *Start the cluster*, we will install and start etcd. The second part, *Make it more secure*, is not mandatory but **highly** suggested.
```

Comment:
That could be done but since it's not very big and still part of setting up the cluster then I think it would just make the guide too scattered without bringing much organization.

Reply:
Fair enough